src/test/ui/for-loop-while/label_break_value_invalid.rs

```diff
@@ -20,14 +20,11 @@ fn lbv_macro_test_hygiene_respected() {
     macro_rules! mac3 {
```

Comment:
mark file as `check-pass`

Reply:
This file still errs on undeclared labels at lines 7 and 29.